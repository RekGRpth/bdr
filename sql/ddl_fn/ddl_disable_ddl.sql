SELECT bdr.bdr_replicate_ddl_command($DDL$ ALTER DATABASE regression RESET bdr.permit_ddl_locking; $DDL$);
SELECT bdr.bdr_replicate_ddl_command($DDL$ ALTER DATABASE postgres RESET bdr.permit_ddl_locking; $DDL$);
