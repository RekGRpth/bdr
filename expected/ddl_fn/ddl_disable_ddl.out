SELECT bdr.bdr_replicate_ddl_command($DDL$ ALTER DATABASE regression RESET bdr.permit_ddl_locking; $DDL$);
 bdr_replicate_ddl_command 
---------------------------
 
(1 row)

SELECT bdr.bdr_replicate_ddl_command($DDL$ ALTER DATABASE postgres RESET bdr.permit_ddl_locking; $DDL$);
 bdr_replicate_ddl_command 
---------------------------
 
(1 row)

